src/main/java/nl/opengeogroep/safetymaps/server/stripes/GeoserverActionBean.java

```diff
@@ -0,0 +1,145 @@
+package nl.opengeogroep.safetymaps.server.stripes;
+
+import static nl.opengeogroep.safetymaps.server.db.JsonExceptionUtils.logExceptionAndReturnJSONObject;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Base64;
+import java.util.Enumeration;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.dbutils.ResultSetHandler;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import net.sourceforge.stripes.action.ActionBean;
+import net.sourceforge.stripes.action.ActionBeanContext;
+import net.sourceforge.stripes.action.ErrorResolution;
+import net.sourceforge.stripes.action.Resolution;
+import net.sourceforge.stripes.action.StreamingResolution;
+import net.sourceforge.stripes.action.StrictBinding;
+import net.sourceforge.stripes.action.UrlBinding;
+import net.sourceforge.stripes.validation.Validate;
+import nl.opengeogroep.safetymaps.server.db.DB;
+
+/**
+ * @author Safety C&T
+ */
+@StrictBinding
+@UrlBinding("/viewer/api/geoserver/{path}")
+public class GeoserverActionBean implements ActionBean {
+    private ActionBeanContext context;
+
+    private static final Log log = LogFactory.getLog(GeoserverActionBean.class);
+
+    @Validate
+    private String path;
+
+    @Override
+    public ActionBeanContext getContext() {
+        return context;
+    }
+
+    @Override
+    public void setContext(ActionBeanContext context) {
+        this.context = context;
+    }
+
+    public String getPath() {
+        return path;
+    }
+
+    public void setPath(String path) {
+        this.path = path;
+    }    
+
+    public Resolution api() {
+        try(Connection c = DB.getConnection()) {
+            if(path != null) {
+                String geoserverUrl = handleParams(getContext().getRequest()).replace("&path=" + path + "&url=", "");
+                URL url = new URL (geoserverUrl);
+                HttpURLConnection connection = (HttpURLConnection) url.openConnection();
```

Comment:
Security: this is an open proxy: /viewer/api/geoserver/get?url=https://www.google.com/